alpha/alpha-server/src/main/java/org/apache/servicecomb/pack/alpha/server/GrpcStartable.java

```diff
@@ -63,15 +64,21 @@ public GrpcStartable(GrpcServerConfig serverConfig, EventBus eventBus, BindableS
       if(unusedPort.isPresent()){
         if (serverConfig.isSslEnable()){
           serverBuilder = NettyServerBuilder.forAddress(
-                  new InetSocketAddress(serverConfig.getHost(), unusedPort.getAsInt()));
+                  new InetSocketAddress(serverConfig.getHost(), unusedPort.getAsInt()))
```

Comment:
I  think it could be better to provide a method here to create ServerBuilder.

Reply:
done